fluent-package/msi/assets/fluent-package-post-migration.bat

```diff
@@ -0,0 +1,66 @@
+@echo on
+title Fluent-package post migration script
+
+if exist "%~dp0..\..\td-agent" (
+  setlocal enabledelayedexpansion
+  sc query state=inactive | findstr fluentdwinsvc
+  if !ERRORLEVEL! == 1 (
+    echo "Active fluentdwinsvc service is detected"
+    net stop fluentdwinsvc
+  )
+  endlocal
+
+  echo "Migrate c:\opt\td-agent\etc\td-agent.conf"
+  if exist "%~dp0..\..\td-agent\etc\td-agent\td-agent.conf" (
+     move /Y "%~dp0..\..\td-agent\etc\td-agent\td-agent.conf" "%~dp0..\etc\fluent\td-agent.conf"
+     echo "Migrate td-agent.conf to c:\opt\fluent\etc\fluent\fluentd.conf"
+     copy /Y "%~dp0..\etc\fluent\td-agent.conf" "%~dp0..\etc\fluent\fluentd.conf"
+  )
+  echo "Create c:\opt\fluent\var\log\fluent directory"
+  if not exist "%~dp0..\var\log\fluent" (
+     mkdir "%~dp0..\var\log\fluent"
+  )
+  echo "Migrate c:\opt\td-agent\var\log\td-agent\* files"
+  for %%f in (%~dp0..\..\td-agent\var\log\td-agent\*) do (
+     move /Y %%f "%~dp0..\var\log\fluent\"
+  )
+  echo "Migrate c:\opt\td-agent\var\log\td-agent\* directory (e.g buffer)"
+  for /d %%d in (%~dp0..\..\td-agent\var\log\td-agent\*) do (
+     move /Y %%d "%~dp0..\var\log\fluent\"
+  )
+  echo "Migrate c:\opt\td-agent\etc\td-agent\* files"
+  for %%f in (%~dp0..\..\td-agent\etc\td-agent\*) do (
+     move /Y %%f "%~dp0..\etc\fluent\"
+  )
+  echo "migrate c:\opt\td-agent\etc\plugin directory"
+  for /d %%d in (%~dp0..\..\td-agent\etc\plugin\*) do (
+     move /Y %%d "%~dp0..\etc\fluent\plugin\"
+  )
+  echo "Migrate c:\opt\td-agent\*.log files"
+  for %%f in (%~dp0..\..\td-agent\*.log) do (
+     move /Y %%f "%~dp0..\fluent\"
+  )
+  echo "Ensure remaining files under td-agent"
+  tree %~dp0..\..\td-agent
+
+  @rem ensure to remove remained c:\opt\td-agent directory
+  setlocal enabledelayedexpansion
+  @rem check whether directory is empty or not.
+  dir /a:-d %~dp0..\..\td-agent
+  if !ERRORLEVEL! == 0 (
+    echo "Deleting empty c:\opt\td-agent"
+    rmdir /S /Q %~dp0..\..\td-agent
+  ) else {
+    echo "Usually empty directory is expected, If not, preserving %~dp0..\..\td-agent as a archive"
+    move /Y %~dp0..\..\td-agent %~dp0..\..\td-agent.old
+  )
+  endlocal

```

Comment:
Somehow, this doesn't seem to work.
I'm investigating it now.

Reply:
I have commented on the cause below.
However, if we should keep the old log files in `td-agent.old` without moving them, we should always create this directory. (commented above)